src/coreclr/tools/aot/Mono.Linker.Tests/TestCasesRunner/ILCompilerDriver.cs

```diff
@@ -82,6 +86,10 @@ public ILScanResults Trim (ILCompilerOptions options, ILogWriter logWriter)
 				options.AdditionalRootAssemblies.ToArray (),
 				options.TrimAssemblies.ToArray ());
 
+			PInvokeILEmitterConfiguration pinvokePolicy = new ILCompilerTestPInvokePolicy ();
```

Comment:
I guess more than feedback, I'm not very familiar with what the InteropStubManager does and why the ILCompilerTestPInvokePolicy looks like not an actual implementation of the method. My guess is that if you hook up the entire framework you need minimal support for interop and this is achieving that (?)

Reply:
You're spot on.
We can technically hit it with the test-only compilation as well, we just didn't do yet. Basically you need to have a policy if there's a PInvoke compiled. With framework that's obviously always the case. With our tests... rarely.
Ultimately we don't care because the ILScanner won't do anything with it really, so we just need the simplest implementation.